audmath/core/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pandas as pd
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def add_pd(doctest_namespace):
+    doctest_namespace["pd"] = pd
```

Comment:
```suggestion
    doctest_namespace['pd'] = pd
```

Indeed a nice solution! :)